lte/gateway/fabfile.py

```diff
@@ -158,71 +156,32 @@ def package(
             _copy_out_c_execs_in_magma_vm(c_gw)
 
 
-@task
-def openvswitch(c, destroy_vm=False, destdir='~/magma-packages/'):
-    # If a host list isn't specified, default to the magma vagrant vm
-    with vagrant_connection(c, 'magma', destroy_vm=destroy_vm) as c_gw:
-        c_gw.run('~/magma/third_party/gtp_ovs/ovs-gtp-patches/2.15/build.sh ' + destdir)
-
-
-@task
-def depclean(c):
-    '''Remove all generated packaged for dependencies'''
-    # If a host list isn't specified, default to the magma vagrant vm
-    with vagrant_connection(c, 'magma') as c_gw:
-        c_gw.run('rm -rf ~/magma-deps')
```

Comment:
Hi, "undocumented" means here that the task is not documented in a `.md` (inc docusaurus) or in the wiki. If someone is using this task and is missing it, then it can be easily be re-added (with a sufficient documentation). If there is a remove task that you are actually using, let me know then I can re-add it. But imo it is not a good pattern to keep code that **could** potentially be useful. This code already caused maintenance effort by being touched during the fabric upgrade - without really be tested, because it is not running in CI.